### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by garyb**
For purescript/purescript#1038

### CODE REVIEW

File: src/Test/QuickCheck.purs [hunk 1]
```diff
@@ -32,42 +33,42 @@ type QC a = forall eff. Eff (trace :: Trace, random :: Random, err :: Exception
...
+quickCheckPure :: forall prop. (Testable prop) => Int -> Int -> prop -> [Result]
+quickCheckPure s = quickCheckPure' { newSeed: s, size: fromNumber 10 } where
   quickCheckPure' st n prop = evalGen (go n) st
     where
-    go n | n <= 0 = return []
+    go n | n <= zero = return []
     go n = do
       result <- test prop
-      rest <- go (n - 1)
+      rest <- go (n - one)
```
Comment (paf31) [inline src/Test/QuickCheck.purs]: Could use `replicateM` here now.

Comment (garyb) [inline src/Test/QuickCheck.purs]: :+1:

File: src/Test/QuickCheck/Gen.purs [hunk 2]
```diff
@@ -103,33 +105,33 @@ frequency x xs = let
     pick n d [] = d
     pick n d ((Tuple k x) : xs) = if n <= k then x else pick (n - k) d xs
   in do
-    n <- chooseInt 1 total
+    n <- choose 0 total
```
Comment (garyb) [inline src/Test/QuickCheck/Gen.purs]: What do you think of this?

`chooseInt` doesn't quite seem right, as the weights are not integers, likewise starting at 1 seems too large a number, as the combined weights might not add up to 1 even. Unless I've misunderstood.

Comment (paf31) [inline src/Test/QuickCheck/Gen.purs]: Yes, I think we should use `choose`. That makes the most sense to me.

**Comment by jdegoes** [comment]
:+1: @garyb I hope I can get this same PR for StrongCheck, too. :)

**Pull request merged**